From e893969dcbf775260ece15e6dd37938d234b45c3 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 3 Nov 2015 14:41:12 +0000
Subject: [PATCH 193/462] Mantis 0037350: integration of EZ-Access device for
 KIS kiosk

added libboost_program_options-vc100-mt-1_54.lib to LIB_BOOST


git-svn-id: svn://localhost/SelfServiceBuild/trunk@295 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index cb25358..8db8bdb 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -261,7 +261,7 @@ goto END_USE_BOOST
   set BOOST_VERSION=1.54
   set DIR_BOOST=%DIR_GLOBALSLIB%\boost\boost_1_54
   set INC_BOOST=-I%DIR_BOOST%
-  set LIB_BOOST=%DIR_BOOST%\lib\libboost_regex-vc100-mt-1_54.lib %DIR_BOOST%\lib\libboost_filesystem-vc100-mt-1_54.lib %DIR_BOOST%\lib\libboost_system-vc100-mt-1_54.lib
+  set LIB_BOOST=%DIR_BOOST%\lib\libboost_regex-vc100-mt-1_54.lib %DIR_BOOST%\lib\libboost_filesystem-vc100-mt-1_54.lib %DIR_BOOST%\lib\libboost_system-vc100-mt-1_54.lib %DIR_BOOST%\lib\libboost_program_options-vc100-mt-1_54.lib
 
   set DIR_REGEX=%DIR_BOOST%
   set INC_REGEX=-I%DIR_REGEX%
-- 
2.41.0.windows.1

